internal/witness/cmd/witness/internal/witness/witness.go

```diff
@@ -51,11 +57,23 @@ func New(wo Opts) (*Witness, error) {
 	// Create the sths table if needed.
 	_, err := wo.DB.Exec(`CREATE TABLE IF NOT EXISTS sths (logID BLOB PRIMARY KEY, sth BLOB)`)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to create table: %v", err)
+	}
+	// Parse the PEM-encoded secret key.
+	p, _ := pem.Decode([]byte(wo.PrivKey))
+	if p == nil {
+		return nil, fmt.Errorf("no PEM block found in %s", wo.PrivKey)
+	}
+	sk, err := x509.ParsePKCS8PrivateKey(p.Bytes)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse private key: %v", err)
+	}
+	if reflect.TypeOf(sk).Kind() == reflect.Ptr {
```

Comment:
🙈

Reply:
This one's _definitely_ worth a comment :)

Reply:
Sigh.  Added one pointing my finger elsewhere in the repo.